app/controllers/foreman_tasks/react_controller.rb

```diff
@@ -1,11 +1,17 @@
 module ForemanTasks
   class ReactController < ::ApplicationController
-    skip_before_action :authorize
```

Comment:
Added authorization, to make tests green, but it also probably make sense to not expose it without having tasks permissions (although the permissions will be covered by API)